javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,146 @@
+/*freecodecamp.org account name: mehmetnuricavdar
+https://www.freecodecamp.org/mehmetnuricavdar
+*/
+
+// ######    Flight booking fullname function
+
+function getFullname(firstname, surname) {
+  const fullName = `${firstname} ${surname}`;
+  return fullName;
+}
+
+let fullname1 = getFullname("Benjamin", "Hughes");
+let fullname2 = getFullname("Mehmet", "Cavdar");
+
+// ###### Formal fullname
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  const fullName = `${firstname} ${surname}`;
+  if (useFormalName === true) {
+    if (gender === "male") {
+      return `Lord ${fullName}`;
+    } else {
+      return `Lady ${fullName}`;
+    }
+  } else {
+    return fullName;
+  }
+}
+getFullname("Benjamin", "Hughes", true, "male"); // returns "Lord Benjamin Hughes"
+getFullname("Benjamin", "Hughes", false); // returns "Benjamin Hughes"
+getFullname("Benjamin", "Hughes", true, "female"); // returns "Lady Benjamin Hughes"
+
```

Comment:
How you can enable your function to handle empty names? imagine I call your function like this `getFullname("", "", true, "male")`

Reply:
i did the necessary changes